Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Goblin Ethnicity

## Changelog:
- Created a custom ethnicity for Goblins using `darkspear_trollish` ethnicity as a template.
- Added a set of hairstyles for Goblins.
- Added localization for Goblin and Gilblin ethnicities.


# How to test:
There are two ways you can test this:
1. Open up Ruler Designer and generate male and female Goblins to see what they look like. The ethnicities are currently called **_Goblin_** and **_Gilblin_**.
2. Alternatively, load up into a game and look for Goblins (there are no Gilblins in the game yet) in-game by using the **_Character Finder_** panel.

Provide feedback on the following:
1. The overall look of the ethnicity.
2. The set of possible hairstyles for goblins. _Are there any that should be added or removed?_

❗ **Important**: The Gilblin ethnicity is missing 3D assets integral to their look, such as custom ears and webbed fingers. But since none currently exist in the available bookmarks, it's not a priority to add them anytime soon. That said, I've done my best to give them a decent placeholder appearance.
